docs/static/winlogbeat-modules.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[winlogbeat-modules]]
+
+== Working with {winlogbeat} Modules
+
+{winlogbeat} comes packaged with pre-built
+{winlogbeat-ref}/winlogbeat-modules.html[modules] that contain the configurations
+needed to collect, parse, enrich, and visualize data from various Windows logging
+providers. Each {winlogbeat} module consists of one or more filesets that contain
+ingest node pipelines, {es} templates, {winlogbeat} input configurations, and
+{kib} dashboards.
+
+You can use {winlogbeat} modules with {ls}, but you need to do some extra setup.
+The simplest approach is to <<use-winlogbeat-ingest-pipelines,set up and use the ingest
+pipelines>> provided by {winlogbeat}.
+
```

Comment:
```suggestion

[discrete]
```

Reply:
@efd6 We're moving toward having more content on a single page so the user doesn't have to click as often.  If you'd like to see your content look like this, adding a [discrete] tag will keep the page intact. 

<img width="859" alt="Screen Shot 2022-09-07 at 7 51 11 PM" src="https://user-images.githubusercontent.com/35154725/189002573-657fe813-c34d-4ade-8745-668e963b3b3d.png">

Reply:
If you like this approach, we can make the corresponding change to filebeat module info.

Reply:
Thanks, yes I think this is better and consistency is also good if that is what we are doing elsewhere.